packages/components/FocusZone/macos/RCTFocusZone.m

```diff
@@ -18,53 +18,26 @@
 // enumerated in the same row (or column) as the current focused view
 static const CGFloat FocusZoneBuffer = 3;
 
-@implementation RCTFocusZone
-
-static inline CGFloat GetDistanceBetweenPoints(NSPoint point1, NSPoint point2)
-{
-	NSPoint delta = NSMakePoint(point1.x - point2.x, point1.y - point2.y);
-	return sqrt(delta.x * delta.x + delta.y * delta.y);
-}
-
-static inline CGFloat GetDistanceBetweenRects(NSRect rect1, NSRect rect2)
-{
-	// Get the top left corner of the rect, top right in RTL
-	bool isRTL = [[RCTI18nUtil sharedInstance] isRTL];
-
-	CGFloat rect1Offset = isRTL ? rect1.size.width : 0;
-	CGFloat rect2Offset = isRTL ? rect2.size.width : 0;
-	
-	NSPoint rect1Corner = NSMakePoint(rect1.origin.x + rect1Offset , rect1.origin.y);
-	NSPoint rect2Corner = NSMakePoint(rect2.origin.x + rect2Offset , rect2.origin.y);
-
-	return GetDistanceBetweenPoints(rect1Corner, rect2Corner);
-}
-
-static inline CGFloat GetMinDistanceBetweenRectVerticesAndPoint(NSRect rect, NSPoint point)
+BOOL ShouldFocusOnView(NSView *view, BOOL forceFocus)
```

Comment:
Can we access `forceFocus` with [self forceFocus] instead? This is to avoid adding unnecessary function parameters.

Reply:
Doing so would make this no longer a static function, and would lead to a bunch of code that looks like this:
`[self firstFocusableViewWithin:self]` since I'd have to make other previously static C methods into instance methods. 

After talking with @toddha , I went with keeping this static to match the rest of this file.